tests/test_wrappers.py

```diff
@@ -745,34 +746,27 @@ def test_prebuilt_model(self):
             estimator = model(build_fn=keras_model)
             check(estimator, loader)
 
-    def test_uncompiled_prebuilt_model_raises_error(self):
-        """Tests that an uncompiled model cannot be used as build_fn param."""
-
-        for config in [
-            "MLPRegressor",
-            "MLPClassifier",
-            "CNNClassifier",
-            "CNNClassifierF",
-        ]:
-            loader, model, build_fn, _ = CONFIG[config]
-            data = loader()
-            x_train, y_train = data.data[:100], data.target[:100]
-
-            n_classes_ = np.unique(y_train).size
-            # make y the same shape as will be used by .fit
-            if config != "MLPRegressor":
-                y_train = to_categorical(y_train)
-                keras_model = build_fn(
-                    X=x_train, n_classes_=n_classes_, n_outputs_=1
-                )
-            else:
-                keras_model = build_fn(X=x_train, n_outputs_=1)
```

Comment:
This test was originally added because the failure from using a prebuilt model without training was unintuitive. These changes make the failure a clean error message from Keras, so I see no need to test for it anymore.